build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -6,6 +6,32 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="xunit" Version="$(XUnitVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
+    <PackageReference Include="xunit.analyzers" Version="$(xunitanalyzersVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
+    <PackageReference Include="xunit.runner.visualstudio" Version="$(XUnitRunnerVisualStudioVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
+    <PackageReference Include="xunit.runner.console" Version="$(XUnitRunnerConsoleVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
```

Comment:
❓ What does this add?

Reply:
It's supposed to be there so that tests are runnable from VS test explorer. I'd love to remove it, if that's not necessary.

Reply:
It has no impact on test explorer for xUnit.net testing.

Reply:
I'll double check and remove it if it works.

Reply:
Yeah, running tests works withotu this thing. Woot. Killing it with fire!